sidebarsDocs.js

```diff
@@ -23,528 +23,652 @@ const sidebars = {
+      {
+        type: 'category',
+        label: 'Clustering',
+        link: {type: 'doc', id: 'clustering'},
+        items: [
+          {
+            type: 'doc',
+            id: 'cluster-formation',
+            label: 'Cluster Formation and Peer Discovery',
+          },
+        ]
+      },
+      {
+        type: 'category',
+        label: 'Working with Plugins',
+        items: [
+          {
+            type: 'doc',
+            id: 'installing-plugins',
+            label: 'Installing Plugins',
+          },
+          {
+            type: 'doc',
+            id: 'plugin-development',
+            label: 'Plugin Development',
+          },
+          {
+            type: 'category',
+            label: 'Management Plugin',
+            link: {type: 'doc', id: 'management'},
+            items: [
+              {
+                type: 'doc',
+                id: 'management-cli',
+                label: 'Management CLI',
+              },
+            ],
+          },
+          {
+            type: 'category',
+            label: 'Federation Plugin',
+            link: {type: 'doc', id: 'federation'},
+            items: [
+              {
+                type: 'doc',
+                id: 'federated-exchanges',
+                label: 'Federated Exchanges',
+              },
+              {
+                type: 'doc',
+                id: 'federated-queues',
+                label: 'Federated Queues',
+              },
+              {
+                type: 'doc',
+                id: 'federation-reference',
+                label: 'Federation Reference',
+              },
+            ],
+          },
+          {
+            type: 'category',
+            label: 'Shovel Plugin',
+            link: {type: 'doc', id: 'shovel'},
+            items: [
+              {
+                type: 'doc',
+                id: 'shovel-static',
+                label: 'Configuring Static Shovels',
+              },
+              {
+                type: 'doc',
+                id: 'shovel-dynamic',
+                label: 'Configuring Dynamic Shovels',
+              },
+            ],
+          }, 
+          {
+            type: 'doc',
+            id: 'stream',
+            label: 'Stream Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'stomp',
+            label: 'STOMP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'web-stomp',
+            label: 'Web STOMP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'mqtt',
+            label: 'MQTT Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'web-mqtt',
+            label: 'Web MQTT Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'ldap',
+            label: 'LDAP Plugin',
+          },
+          {
+            type: 'doc',
+            id: 'oauth2',
+            label: 'OAuth 2.0 Authentication/Authorisation Backend Plugin',
```

Comment:
This one is already linked in the authentication section later. You can drop this entry.

Reply:
Done, thanks